lib/elastic_apm/error_builder.rb

```diff
@@ -19,10 +20,13 @@ def build_exception(exception, handled: true)
 
       if (transaction = ElasticAPM.current_transaction)
         error.context = transaction.context.dup
+        error.trace_id = transaction.trace_id
+        error.parent_id = transaction.id
```

Comment:
The parent of an `error` could also be a `span`.  There's a dedicated `transaction_id` defined for an error, that should be set if available. See [json spec](https://github.com/elastic/apm-server/blob/v2/docs/spec/errors/v2_error.json#L21).